### PR TITLE
Bump wrapper and fix resulting deprecation warnings

### PR DESCRIPTION
**Pull request by marcphilipp**

### CODE REVIEW

File: buildSrc/subprojects/integration-testing/src/main/kotlin/org/gradle/gradlebuild/test/integrationtests/DistributionTestingPlugin.kt [hunk 1]
```diff
@@ -78,13 +79,14 @@ class DistributionTestingPlugin : Plugin<Project> {
     }
 
     private
-    fun DistributionTest.configureGradleTestEnvironment(providers: ProviderFactory, layout: ProjectLayout, basePluginConvention: BasePluginConvention) {
+    fun DistributionTest.configureGradleTestEnvironment(providers: ProviderFactory, layout: ProjectLayout, basePluginConvention: BasePluginConvention, objects: ObjectFactory) {
```
Comment (wolfs) [inline buildSrc/subprojects/integration-testing/src/main/kotlin/org/gradle/gradlebuild/test/integrationtests/DistributionTestingPlugin.kt]: Do we still need the `layout` here? Can we use the `ObjectFactory` to create `providers`, too?

Comment (oehme) [reply buildSrc/subprojects/integration-testing/src/main/kotlin/org/gradle/gradlebuild/test/integrationtests/DistributionTestingPlugin.kt]: Right, seems we've added the directory/file property methods to `ObjectFactory` now.

Comment (marcphilipp) [reply buildSrc/subprojects/integration-testing/src/main/kotlin/org/gradle/gradlebuild/test/integrationtests/DistributionTestingPlugin.kt]: We need `layout` for `layout.projectDirectory` and `layout.buildDirectory` and `ObjectFactory` cannot create `Providers`.

**Comment by oehme** [comment]
@wolfs already said everything :)

**Pull request merged**